src/data_table/local_data_table.js

```diff
@@ -29,6 +29,11 @@ class LocalDataTable extends View {
     _.extend(this, _.pick(this.options, ["selectedIds", "paginate"]));
     _.bindAll(this, "_onRowCreated", "_onBulkHeaderClick", "_onBulkRowClick", "_bulkCheckboxAdjust", "_onDraw",
       "_onColumnVisibilityChange", "_onColumnReorder");
+    if (this.options.urlPagination !== undefined) {
+      this.urlPagination = this.options.urlPagination;
```

Comment:
per other comment, this would be ` = this.pagination && this.options.urlPagination;`